Dockerfile

```diff
@@ -1,20 +1,19 @@
-FROM centos:centos8
+FROM centos/ruby-27-centos7:2.7
+
+USER 0
+
+RUN yum update -y \
+    && yum install -y \
+    libxml2-devel \
+    libxml2-devel \
+    perl-Devel-Peek \
+    perl-FreezeThaw \
+    perl-HTML-Parser \
+    perl-libwww-perl \
+    wget \
+    && yum clean all
 
 WORKDIR /headerdoc_build
-RUN dnf update -y \
-&& dnf install -y epel-release \
-&& dnf groupinstall -y "Development Tools" \
-&& dnf install -y \
-libxml2-devel \
-perl-HTML-Parser \
-perl-libwww-perl \
-perl-FreezeThaw \
-libxml2-devel \
-wget \
-perl-Devel-Peek \
-ruby-devel \
-rubygem-bundler \
```

Comment:
**thought:** These 2 packages are already included in the `centos/ruby-27-centos7` base image.